### ISSUE TITLE
Minor inconvenience: Laser + Beam weapons: Confusing names.

### ISSUE THREAD

**Issue by Cpt-Fortius** [tsunamayo/Starship-EVO#5858]
### Short Description of the issue:

"Beam" weapon shoots beams. Good.
"Laser" weapon shoots energy projectiles. When someone says "Laser" I think about beams.

### Solution:

"Laser" weapon should be renamed "Blaster" or something similar. It corresponds better to it's function of shooting blasts of energy. Avoids confusion.

**Comment by tsunamayo** [comment]
Yeah I agree I wanted to do that for a while, not sure about blaster though. Will do for the weapon refactor update.

**Comment by Ketsueko** [comment]
Maybe use the term "plasma" canon? unless you want to create a real plasma cannon that use gas as ammunition, i think it could work?
